code/modules/power/power.dm

```diff
@@ -278,7 +278,7 @@
 //dist_check - set to only shock mobs within 1 of source (vendors, airlocks, etc.)
 //No animations will be performed by this proc.
 /proc/electrocute_mob(mob/living/carbon/M, power_source, obj/source, siemens_coeff = 1, dist_check = FALSE)
-	if(ismecha(M.loc))
+	if(!istype(M) || ismecha(M.loc))
```

Comment:
If this is happening, it'd make much more sense to turn this proc into /mob/living/carbon/proc/electrocute

Reply:
you're supposed to be able to zap more than just living mobs, I"m not sure that this is correct.

Reply:
From quick glance this is used in airlock and grille shocks which probably should shock simple animals too. So this is not strictly correct.